style.css

```diff
@@ -0,0 +1,690 @@
+}
+
+.bottom-nav li {
+  color: var(--black-color);
+}
+
+.bottom-nav button {
+  padding: 8px;
+  background-color: transparent;
+  border: 5px solid var(--red-color);
+  color: var(--red-color);
+}
+
+/* headline */
+
+#headline {
+  width: 100%;
+  background-image: linear-gradient(rgba(214, 209, 243, 0.753), rgba(214, 209, 243, 0.753)), url("images/Jesus-mobile.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 120px 0;
+}
+
+.headline-content {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.headline-content h1 {
+  font-size: 40px;
+  color: var(--red-color);
+  text-transform: uppercase;
+  font-weight: 800;
+}
+
+.headline-description {
+  padding: 16px;
+  border: 2px solid var(--white-color);
+  color: var(--black-color);
+}
+
+.headline-footer {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.headline-footer h2 {
+  color: var(--black-color);
+  text-transform: uppercase;
+  font-weight: 900;
+}
+
+.headline-footer h3 {
+  color: gray;
+  font-size: 16px;
+}
+
+/* program  */
+#program {
+  background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNSIgaGVpZ2h0PSIxNSI+CjxyZWN0IHdpZHRoPSI1MCIgaGVpZ2h0PSI1MCIgZmlsbD0iIzI4MjgyOCI+PC9yZWN0Pgo8Y2lyY2xlIGN4PSIzIiBjeT0iNC4zIiByPSIxLjgiIGZpbGw9IiMzOTM5MzkiPjwvY2lyY2xlPgo8Y2lyY2xlIGN4PSIzIiBjeT0iMyIgcj0iMS44IiBmaWxsPSJibGFjayI+PC9jaXJjbGU+CjxjaXJjbGUgY3g9IjEwLjUiIGN5PSIxMi41IiByPSIxLjgiIGZpbGw9IiMzOTM5MzkiPjwvY2lyY2xlPgo8Y2lyY2xlIGN4PSIxMC41IiBjeT0iMTEuMyIgcj0iMS44IiBmaWxsPSJibGFjayI+PC9jaXJjbGU+Cjwvc3ZnPg==");
+}
+
+.program-content {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 95%;
+  padding: 32px 0;
+}
+
+.program-title,
+.testimonial-title,
+.partner-title,
+.logo-title,
+.events-title {
+  padding: 32px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: center;
+}
+
+.program-title h2 {
+  color: var(--grey-color);
+}
+
+.program-title hr,
+.testimonial-title hr,
+.partner-title hr,
+.logo-title hr,
+.events-title hr {
+  border: 1px solid var(--red-color);
+  background-color: var(--red-color);
+  width: 26px;
+}
+
+.program-card {
+  background-color: rgba(60, 63, 70, 0.8);
+  padding: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 10px;
+  line-height: 1.4em;
+  height: 100px;
+  font-size: 12px;
+}
+
+.card-icon {
+  height: 40px;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  width: 40%;
+}
+
+.card-description {
+  width: 50%;
+}
+
+.program-card h3 {
+  color: var(--red-color);
+  white-space: normal;
+  word-break: break-word;
+}
+
+.card-description p {
+  color: var(--grey-color);
+}
+
+.program-btn {
+  width: 100%;
+  text-align: center;
+  margin-top: 24px;
+}
+
+.program-btn button {
+  width: 80%;
+  padding: 24px;
+  color: var(--grey-color);
+  background-color: var(--red-color);
+  font-size: 20px;
+  cursor: pointer;
+  margin-bottom: 60px;
+}
```

Comment:
- [ ] Kindly, readjust the size and spacing of your Join button to make it matches the provided template.
- [ ] Also you may consider readjusting the color of the elemet.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203970029-1e2f0e77-c4d8-441e-bef0-9dd8301c796c.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203970308-92784576-86dd-4635-aefc-9eed4b9eb7df.png)

Reply:
This is my customized selection. Please allow me to keep my design according to my own customized style I desire.